tests/php/test-class-amp-youtube-embed-handler.php

```diff
@@ -41,6 +45,8 @@ class Test_AMP_YouTube_Embed_Handler extends WP_UnitTestCase {
 	 */
 	public function setUp() {
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).